README.md

```diff
@@ -1,187 +1,130 @@
+# go to WP plugins directory
+$ cd path_to_wp_directory/wp-content/plugins
+# clone this repository
+$ git clone https://github.com/mailpoet/mailpoet.git
+$ cd mailpoet
+# create the .env file
+$ cp .env.sample .env
+# change the values on .env file
+# download composer
 $ curl -sS https://getcomposer.org/installer | php
+$ chmod +x ./composer.phar
+# install PHP dependencies
 $ ./composer.phar install
-```
-
-- Install dependencies.
-```sh
+# install all dependencies (PHP and JS)
 $ ./do install
-```
-
-- Update dependencies when needed.
-```sh
-$ ./do update
-```
-
-- Copy .env.sample to .env.
-```sh
-$ cp .env.sample .env
-```
-
-- Compile assets.
-```sh
+# compile JS and CSS files
 $ ./do compile:all
 ```
 
-# Tests.
-
-- Unit tests (using [verify](https://github.com/Codeception/Verify)):
-```sh
-$ ./do test:unit
-```
-
-- JS tests (using Mocha):
-```sh
-$ ./do test:javascript
-```
-
-- Debug tests:
-```sh
-$ ./do test:debug
-```
-
-- Code linters and quality checkers:
-```sh
-$ ./do qa
-```
-
-- Javascript linter:
-```sh
-$ ./do lint:javascript
-```
-
-# CSS
-- [Stylus](https://learnboost.github.io/stylus/)
-- [Nib extension](http://tj.github.io/nib/)
-
-```sh
-assets/css/src -> place your *.styl files here
-```
-
-### Watch for changes and recompile
-```sh
-$ ./do watch
-```
-
-## Module loading and organization
-
-Our JS modules are stored in `assets/js/` folder. Modules should follow AMD module definition style:
-
-```js
-define('moduleName', ['dependency1', 'dependency2'], function(dependency1, dependency2){
-  // Module code here
-
-  return {
-    // Module exports here
-  };
-})
-```
-
-Module loader will look for `dependency1` in `node_modules/` dependencies, as well as in `assets/js`. So you can use dependencies, defined in `package.json`, without the need of providing an absolute path to it.
-Once found, dependencies will be injected into your module via function arguments.
-
-When it comes to loading modules on a real page, WebPack uses "entry points" to create different bundles. In order for the module to be included in a specific bundle, it must be reachable from that bundle's entry point. [A good example on WebPack's website](http://webpack.github.io/docs/code-splitting.html#split-app-and-vendor-code).
-
-Once javascript is compiled with `./do compile:javascript`, your module will be placed into a bundle. Including that bundle in a webpage will give provide you access to your module.
-
-## Handlebars (`views/*.hbs`)
-
-```html
-<!-- use the `templates` block -->
-<% block templates %>
-  <!-- include a .hbs template -->
-  <%= partial('my_template_1', 'form/templates/toolbar/fields.hbs') %>
-
-  <!-- include a .hbs template and register it as a partial -->
-  <%= partial('my_template_2', 'form/templates/blocks.hbs', '_my_partial') %>
-
-  <!-- custom partial using partial defined above -->
-  <script id="my_template_3" type="text/x-handlebars-template">
-    {{> _my_partial }}
-  </script>
-<% endblock %>
-```
-
-# i18n
```

Comment:
I don't think we should delete all this information about translation functions. I think we should expand this section and explain what and when to use.

Reply:
I agree that we should keep the section about translation.